protocols/epic.js

```diff
@@ -126,9 +126,6 @@ export default class Epic extends Core {
       throw new Error('Server not found')
     }
 
-    state.name = desiredServer.attributes.CUSTOMSERVERNAME_s
-    state.map = desiredServer.attributes.MAPNAME_s
-    state.password = desiredServer.attributes.SERVERPASSWORD_b
```

Comment:
These could remain here and only overwrite the values if needed (MAPNAME_s is the same for asa and this one).

Reply:
In this case it is `MAP_NAME_s`, in the ASA it is `MAPNAME_s`. I thought the epic would be a little more stable in that sense, but it seems that most of the attributes are game dependent.

Reply:
Oh, oops, well they could be default values then.